blog/2015-03-16.Debugging-an-Externally-Launched-Wildfly.adoc

```diff
@@ -0,0 +1,97 @@
+= Debugging an Externally launched WildFly
```

Comment:
I added the following in the PR:


+While the changes you'll read here today apply to all running java applications, I'll be focussing on debugging an externally launched WildFly for this article.